docs/guide/getting-started.md

```diff
@@ -23,23 +23,24 @@ Make sure you have installed Helm 3
 ```
 $ helm install edgemesh \
 --set server.nodeName=<your node name> \
---set server.publicIP=<your node eip> \
+--set "server.advertiseAddress=<your edgemesh server adveritise address list, such as node eip>" \
 https://raw.githubusercontent.com/kubeedge/edgemesh/main/build/helm/edgemesh.tgz
 ```
 
-server.nodeName specifies the node deployed by edgemesh-server, and server.publicIP specifies the public IP of the node. The server.publicIP can be omitted, because edgemesh-server will automatically detect and configure the public IP of the node, but it is not guaranteed to be correct.
+server.nodeName specifies the node deployed by edgemesh-server, and server.advertiseAddress specifies the EdgeMesh-Server advertise address list. 
```

Comment:
edgemesh-server or EdgeMesh-Server? We need to unify them.

Reply:
I think edgemesh-server would be better